### PR TITLE
[GitLab Critical Security Release] Upgrade to GitLab CE 15.8.1

### PR DESCRIPTION
**Pull request by rubensilva09**
GitLab Patch Release: 15.8.1 (see https://about.gitlab.com/releases/2023/01/31/security-release-gitlab-15-8-1-released/)

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -2,15 +2,21 @@
...
 
-**15.8.0**
+**15.8.1**
 
-- gitlab: upgrade CE to v15.8.0
-- gitaly: upgrade to v15.8.0
-- gitlab-pages: upgrade to v15.8.0
+- gitlab: upgrade CE to v15.8.1
+- gitaly: upgrade to v15.8.1
+- gitlab-pages: upgrade to v15.8.1
```
Comment (sachilles) [inline Changelog.md:19]: Please revert the changes you made, since this version numbers have to be 15.8.0!

File: Dockerfile [hunk 2]
```diff
@@ -1,14 +1,14 @@
 FROM ubuntu:focal-20221130
```
Comment (sachilles) [inline Dockerfile:1]: Could you be so kind and use `ubuntu:focal-20230126` as the underlying base image?

**Review by sachilles — CHANGES_REQUESTED** [review]
Please submit the changes as requested! In any case thanks for your contribution.

**Review by sachilles — APPROVED** [review]
Now, it looks good to me.

**Comment by antt1995** [comment]
@sachilles 
Change Log for Focal Update?

File: Changelog.md [hunk 3]
```diff
@@ -2,6 +2,12 @@
 
 This file only reflects the changes that are made in this image. Please refer to the upstream GitLab [CHANGELOG](https://gitlab.com/gitlab-org/gitlab-foss/blob/master/CHANGELOG.md) for the list of changes in GitLab.
 
+**15.8.1**
+
+- gitlab: upgrade CE to v15.8.1
+- gitaly: upgrade to v15.8.1
+- gitlab-pages: upgrade to v15.8.1
+
```
Comment (antt1995) [inline Changelog.md:11]: - ubuntu: upgrade to focal-20230126

Comment (sachilles) [reply Changelog.md:11]: Thanks! I realized the missing bullet.

**Comment by Re4zOon** [comment]
Hi,

Can you build the image quickly? :)
I wanna go home, but first I would like to update our gitlab :D
